odxtools/multiplexer.py

```diff
@@ -149,16 +149,8 @@ def __repr__(self) -> str:
 
 def read_switch_key_from_odx(et_element):
     """Reads a Switch Key for a Multiplexer."""
-    byte_position = (
-        int(et_element.find("BYTE-POSITION").text)
-        if et_element.find("BYTE-POSITION") is not None
-        else None
-    )
-    bit_position = (
-        int(et_element.find("BIT-POSITION").text)
-        if et_element.find("BIT-POSITION") is not None
-        else 0
-    )
+    byte_position = int(et_element.findtext("BYTE-POSITION"))
+    bit_position = int(et_element.findtext("BIT-POSITION", 0))
```

Comment:
bit position is optional, int(None) will fail

Reply:
while new solution will work, it is IMO nicer if a function call always returns objects of the same type, i.e.,
```suggestion
    bit_position = int(et_element.findtext("BIT-POSITION", "0"))
```

a slightly more severe issue with this: if the input file does not specify a bit position, it is impossible to detect that fact anymore, i.e., all output files written must either skip the tag for bit_position == 0 or the tag must always be written.

Reply:
adding a `findint(et_element, location, default=None)` utility function analogous to `findtext()` would be a workable solution. I'm just not sure if that is a good idea, though...

Reply:
reverted the code